src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -0,0 +1,245 @@
+from datetime import datetime
+import json
+import os
+import sas
+import sas_testcase
+from util import winnforum_testcase
+class FullActivityDumpTestcase(sas_testcase.SasTestCase):
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  @classmethod
+  def setUpClass(self):
+    self._sas_server = sas.GetServer()
```

Comment:
What guarantees that two test cases don't interact with one another? Shouldn't this be in setUp() instead?

Reply:
I already deleted the setUpClass function and put startserver and stopserver into util.py file.